common/components/BalanceSidebar/BalanceSidebar.jsx

```diff
@@ -94,10 +99,7 @@ export class BalanceSidebar extends React.Component {
               {!!tokenExplorer &&
                 <li>
                   <a
-                    href={tokenExplorer.address.replace(
-                      '[[address]]',
-                      wallet.getAddress()
-                    )}
+                    href={tokenExplorer.address.replace('[[address]]', address)}
```

Comment:
When considering future networks / chains, we should probably have the ability to not display a token explorer if there isn't one. Not sure best place for it (probably not here :P)